_posts/2015-04-03-april-newsletter.md

```diff
@@ -0,0 +1,250 @@
+---
+layout: newsletter
+title: "Newsletter #5 - TODO"
+category: newsletter
+permalink: /news/2015/april/
+---
+
+Welcome to the fifth newsletter for Neovim.
+
+## General News
+
+### Blackduck 2014 Open Source Rookie
+
+[Blackduck][blackduck-site], a company that helps other corporations in the
+adoption of open source software, chose Neovim as one of the projects to feature
+of this last year. The company annually recognizes some of the top new open
+source projects that started during the year.
+
+### ICCF Holland Donations
+
+Mentioned in the last letter, [Unixstickers][iccf-unixstickers] has added
+[Neovim stickers][iccf-stickers] to their inventory. Every sticker that is
+bought increases the amount donated to [ICCF Holland][iccf-site], Bram
+Moolenaar's charity.
+
+Since they've been added, $106 has been raised which Unixstickers [just
+donated][iccf-tweet] on March 24th.
+
+### Related Projects
+
+The [Neovim Wiki][related-main-wiki] contains a page titled [Related
```

Comment:
What do you mean? The `[related-main-wiki]` is linked with [this line](https://github.com/neovim/neovim.github.io/pull/82/files#diff-e6d5661bcb3c99d1e3afa66e3aa0e12fR280)